lib/src/models/user.dart

```diff
@@ -44,7 +48,36 @@ class User {
         firstName: firstName ?? this.firstName,
         languages: languages ?? this.languages,
         lastName: lastName ?? this.lastName,
+        linkedAccounts: linkedAccounts ?? this.linkedAccounts,
         phoneNumber: phoneNumber ?? this.phoneNumber,
         profiles: profiles ?? this.profiles,
       );
+}
+
+class LinkedAccount {
+  String name;
+  String? accessToken;
+  String? refreshToken;
+  String? externalId;
+
+  LinkedAccount({required this.name, this.accessToken, this.refreshToken, this.externalId,});
+
+  LinkedAccount.fromJson(Map<String, dynamic> json)
+      : name = json['serviceName'],
+        accessToken = json['accessToken'],
+        refreshToken = json['refreshToken'],
+        externalId = json['externalId'];
+
+  LinkedAccount cloneWith({
+    String? name,
+    String? accessToken,
+    String? refreshToken,
+    String? externalId,
```

Comment:
I'm not sure we should expose these values to the app. Do you know if/how they're used?

Reply:
I knew you would ask 😄 ! I should have left a comment there. It is not used AT THE MOMENT. I'll remove them.